app/views/availabilities/places/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container-fluid" style="display: flex; align-items: center; justify-content: center; height: calc(100% - 71px);">
```

Comment:
Um, don't put CSS here

Reply:
All your comments in `app/views/availabilities/places/*` are basically from @lungsin 's commit :sweat_smile:

Reply:
Ummm so? haha

Reply:
Done